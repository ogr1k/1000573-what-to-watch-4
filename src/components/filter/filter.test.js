import React from "react";
import renderer from "react-test-renderer";
import Filter from "./filter.jsx";

const genre = `Action`;
const activeFilter = `Action`;

it(`Render Films-List`, () => {
  const tree = renderer
    .create(< Filter genre={genre} onClick={() => {}} activeFilter={activeFilter}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
