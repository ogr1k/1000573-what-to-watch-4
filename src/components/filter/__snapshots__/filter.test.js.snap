// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Films-List 1`] = `
<li
  className="catalog__genres-item catalog__genres-item--active"
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    Action
  </a>
</li>
`;
